e2e/quick_start_test.go

```diff
@@ -17,6 +17,7 @@ package e2e
 import (
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
```

Comment:
Please add a blank line below.

Reply:
Thx and got it. Wait me to work out a clean way than I fix it.

Reply:
Sure, take your time.